controllers/controller_functions.js

```diff
@@ -27,3 +27,20 @@ if (username.length === 0 || body.length === 0){
 }
 return true;
 };
+
+
+exports.checkIfCategoryExists = (category) => {
```

Comment:
This function looks great - the benefit of a function like this is that you can check why you have an empty array when selecting all reviews with a category that does exist, but just doesn't have any associated reviews

For this reason, at the moment, you could just manually promise.reject after checking if the `rows` array is empty. However, if you were to test for a 200 - empty array of reviews for a category that DOES exist, this would prove very handy!